products-page/products-page.css

```diff
@@ -41,6 +42,36 @@
     flex-direction: column;
     font-size: calc(var(--text-size-m) - 6px);
 }
+
 label{
     padding: 0.25rem;
+}
+
+.products-shown{
+    font-size: var(--text-size-m);
+    padding-bottom: 1rem;
+}
+
+.products-number{
+    font-size: calc(var(--text-size-m) - 8px);
+}
+
+.products{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 25px;
```

Comment:
Can use rem too depending on your requirements

Reply:
Thanks for pointing will change while refactoring.